index.html

```diff
@@ -0,0 +1,164 @@
+           and kelly Wearstler.
+           The school largest graduating class will celebrate with
+            digital and in-person.
+        </p>
+      </div>
+      <div class="headline-start">
+        <h2>2022.8.5(THU) ~ 6(FRI)</h2>
+        <p>@Lorem ipsum dolor sit amet conse ctetur adipi?</p>
+      </div>
+    </section>
+    <section class="program">
+      <h2>Main Program</h2>
+      <hr class="hr" />
+      <div class="card-container">
+        <div class="program-card">
+          <img class="card-img" src="./images/icon_presenter.png" alt="" />
+          <h3 class="card-title">Lecture</h3>
+          <p class="card-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quibusdam,
+            animi?
+          </p>
+        </div>
+        <div class="program-card">
+          <img class="card-img" src="./images/icon_program.png" alt="" />
+          <h3 class="card-title">Exhibition</h3>
+          <p class="card-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quibusdam,
+            animi?
+          </p>
+        </div>
+        <div class="program-card">
+          <img class="card-img" src="./images/icon_messages.png" alt="" />
+          <h3 class="card-title">Forum</h3>
+          <p class="card-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quibusdam,
+            animi?
+          </p>
+        </div>
+        <div class="program-card">
+          <img class="card-img" src="./images/icon_board.png" alt="" />
+          <h3 class="card-title">Ignite</h3>
+          <p class="card-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quibusdam,
+            animi?
+          </p>
+        </div>
+        <div class="program-card">
+          <img class="card-img" src="./images/icon_wheel.png" alt="" />
+          <h3 class="card-title">Workshop</h3>
+          <p class="card-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quibusdam,
+            animi?
+          </p>
+        </div>
+      </div>
+      <button class="btn-join" type="button">CozySpace</button>
+      <a href="#" class="see">SEE THE WHOLE PROGRAMME</a>
+    </section>
+    <section class="featured-speakers">
+      <h2>Featured Speakers</h2>
+      <hr class="hr" />
+      <div class="wrapper" id="wrapper">
+       
+      </div>
+      <button class="see-more" type="button">
+        See More<i class="fas fa-angle-down"></i>
+      </button>
+    </section>
+    <section class="partner">
+      <h2>Partners</h2>
+      <hr class="hr" />
+      <div class="partner-logo">
+          <p>Gensler</p>
+          <p> HBA</p>
+          <p>yabu pushelberg</p>
+          <p>INTERIOR ARCHITECTS</p>
+      </div>
+    </section>
+    <footer>
+      <div class="footer-info">
+        <h5>&#169;CozySpace.</h5>
+        <p>Our design approach thoughtfully engages all components into a holistic solution.</p>
+      </div>
```

Comment:
- [ ] Kindly Fix the last footer in the home section and in the about section because I can see that there is an empty space but it is not exist in the original design.👍🏻✅✅✅

![image](https://user-images.githubusercontent.com/58553711/183259583-d59382e5-7f49-4907-989a-af54a85176f1.png)